src/test/java/org/sonarlint/intellij/config/global/SonarQubeServerTest.java

```diff
@@ -49,18 +43,19 @@ public void testRoundTrip() {
 
   @Test
   public void testEncoded() {
-    server.setPassword("pass");
-    server.setToken("token");
+    SonarQubeServer.Builder builder = SonarQubeServer.newBuilder()
+      .setPassword("pass")
+      .setToken("token");
 
-    String encodedPass = server.getEncodedPassword();
-    String encodedToken = server.getEncodedToken();
-    server.setEncodedPassword(encodedPass);
-    server.setEncodedToken(encodedToken);
+    String encodedPass = builder.getEncodedPassword();
+    String encodedToken = builder.getEncodedToken();
+    builder.setEncodedPassword(encodedPass);
+    builder.setEncodedToken(encodedToken);
 
-    assertThat(server.getEncodedPassword()).isEqualTo(encodedPass);
-    assertThat(server.getEncodedToken()).isEqualTo(encodedToken);
+    assertThat(builder.getEncodedPassword()).isEqualTo(encodedPass);
+    assertThat(builder.getEncodedToken()).isEqualTo(encodedToken);
 
-    assertThat(server.getPassword()).isEqualTo("pass");
-    assertThat(server.getToken()).isEqualTo("token");
+    assertThat(builder.getPassword()).isEqualTo("pass");
+    assertThat(builder.getToken()).isEqualTo("token");
```

Comment:
Are the getters really necessary on the builder? You could `.build()` and assert on the built objects.